src/26-js-ipfs-0-24/index.md

```diff
@@ -0,0 +1,68 @@
+---
+baseurl: ..
+template: tmpl/layouts/post.html
+breadcrumbs:
+  - {name: "26-js-ipfs-0-24", link: "./" }
+id: 26-js-ipfs-0-24
+date: 2017-05-25
+title: js-ipfs 0.24.0 released
+author: David Dias
+collection: posts
+---
+
+I am pleased to announce to everyone in our Community that js-ipfs 0.24.0 has been successfully launched! This new minor release brings new features, bug fixes and new examples so that you can jump in and start hacking your IPFS enabled apps right away!
+
+<blockquote class="twitter-tweet" data-conversation="none" data-lang="en"><p lang="en" dir="ltr"><a href="https://twitter.com/IPFSbot">@IPFSbot</a> Woot! As promissed, new version of js-ipfs released!<br><br>🚀 `ipfs@0.24.0 [18:39] Published to npm.` 🚀<br><br>Release log here: <a href="https://t.co/20O2L2Scq5">https://t.co/20O2L2Scq5</a></p>&mdash; David Dias (@daviddias) <a href="https://twitter.com/daviddias/status/867512323732365312">May 24, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+# Highlights
+
+### 👢 WebSockets DNS Bootstrappers by default on the browser
+
+With this release, you won’t need to connect to Bootstrapper nodes manually, they will dialled from the start through their WebSockets endpoints. This also means that we’ve successfully deploy DNS support for multiaddrs, so that you can host your js-ipfs enabled app behind an HTTPS domain and dial to our Bootstrappers through WSS.
+
+### 🎈WebRTC is now a default transport
+
+We now include WebRTC multiaddr by default on new js-ipfs init calls. This means that both your Node.js and Browser nodes will be able to dial each other using WebRTC and discover other nodes in the network through Signalling Peer Discovery.
+
+Caveat, for Linux/Windows users, due to Node.js wrtc module limited support, you will have to set an environment variable or an Experimental flag to do so, check the README to learn how to do so.
+
+### 🗺 1st Phase of DHT Implementation Complete
+
+We’ve a working DHT implementation, with this implementation, libp2p got simpler and new PeerRouting and ContentRouting API. We are currently holding this feature behind an Experimental flag until we prune out some interoperability issues with go-ipfs. 
```

Comment:
We now have a working DHT implementation, which means that js-libp2p and js-ipfs are now capable of Peer Routing and Content Routing. This feature should be treated as experimental for the time being, and is behind a feature flag until a few remaining interoperability issues are cleared out. [link to info about the feature flag]